### PR TITLE
addpkg: pid_children-git

### PR DESCRIPTION
**Pull request by yan12125**
Closes: https://github.com/archlinuxcn/repo/issues/1439

### CODE REVIEW

File: archlinuxcn/pid_children-git/tmpfiles.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+z /usr/bin/kill_children - - pkg
```
Comment (lilydjwg) [inline archlinuxcn/pid_children-git/tmpfiles.conf]: Shouldn't we specify permissions here? Changing group would discard the sgid bit.

Comment (yan12125) [reply archlinuxcn/pid_children-git/tmpfiles.conf]: Thanks I forgot that is how chown/chgrp usually works. But again systemd does something different - it tries to keep suid/sgid as much as possible (https://github.com/systemd/systemd/blob/v244/src/tmpfiles/tmpfiles.c#L833). Anyway specifying permissions explicitly is better.

Comment (lilydjwg) [reply archlinuxcn/pid_children-git/tmpfiles.conf]: What? It means that I can get root for free if I can find a systemd-managed root owned file that I can unlink and recreate....

**Comment by yan12125** [comment]
Thanks for the comments!

**Pull request merged**